src/main/java/mate/academy/model/Book.java

```diff
@@ -35,4 +40,11 @@ public class Book {
     private String coverImage;
     @Column(name = "is_deleted", nullable = false, columnDefinition = "BIT")
     private boolean isDeleted = false;
+    @ManyToMany(fetch = FetchType.LAZY)
+    @JoinTable(
+            name = "categories",
+            joinColumns = @JoinColumn(name = "book_id"),
+            inverseJoinColumns = @JoinColumn(name = "category_id")
+    )
+    private Set<Category> categories;
```

Comment:
From the task description:
> The `Book` class must have the `private Set<Category> categories = new HashSet<>();` field. 
```suggestion
    private Set<Category> categories = new HashSet<>();
```